cctools/as/arm.c

```diff
@@ -16956,10 +16966,12 @@ md_apply_fix (fixS *	fixP,
     case BFD_RELOC_ARM_OFFSET_IMM:
 #ifdef NOTYET
       if (!fixP->fx_done && seg->use_rela_p)
+	  value = 0;
 #else
-      if (!fixP->fx_done && 0)
+// The goggles do nothing!
```

Comment:
instead of commenting this out, maybe just change the preprocessor conditional that it's in?